spinalcordtoolbox/download.py

```diff
@@ -159,7 +159,13 @@
             "https://github.com/ivadomed/multimodal-registration/releases/download/r20220512/models.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepreg_models"),
-    }
+    },
+    "data_normalize_metric_pam50": {
+        "mirrors": [
+            "https://github.com/spinalcordtoolbox/data_normalize_metric_pam50/archive/refs/tags/r20230221.zip"
```

Comment:
From what I can tell, this link comes from the "Source code (.zip)" download for the [`r20230221`](https://github.com/spinalcordtoolbox/data_normalize_metric_pam50/releases/tag/r20230221) tag.

However, is there merit to instead making a full-fledged release (like we have for other datasets)? And, if we do make a release, should we create a separate `.zip` asset (like we have done for other datasets)?

![image](https://user-images.githubusercontent.com/16181459/220439452-e56a8ca7-66d3-4ab5-b7df-cc28db3c0005.png)

Reply:
For some historical context, NeuroPoly uses (or, used to use) an internally-developed Python package called [`distriploy`](https://github.com/neuropoly/distriploy) to upload zipped archives to GitHub and OSF. That's why you'll sometimes see `.distriploy.yml` files in repos such as `PAM50` and `sct_testing_data`.

Distriploy defaults to making separate `.zip` assets and attaching them to the release (even if they're identical to the Source Code .zip), which is why we see this behavior elsewhere.

Reply:
From what I can tell:

- Separate assets are static, while "Source code (.zip)" will adapt to whatever the state of the tag is.
- Separate assets provide download statistics, while "Source code (.zip)" do not.
- Separate assets are guaranteed to have their checksums preserved, while "Source code (.zip)" archives may change (see [recent controversy](https://news.ycombinator.com/item?id=34586917), though this was rolled back).

I don't know how much we care about any of these pros/cons? Part of me thinks that it's fine to just use the "Source code (.zip)" archives, especially in cases like this where the contents of the repo are _exactly_ what we want to be downloading.

Reply:
> (But, even if we do use the Source code .zips, we still might want to consider an actual release instead of just a tag?)

yes agreed, I actually deleated the relase associated with teh tag, I thought it didn't work!

We can go with the source.zip for now, and if the cons become a problem, we can always change that in the futur